lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -393,13 +387,13 @@ public function get_styles_block_nodes() {
 	 * An internal method to get the block nodes from a theme.json file.
 	 *
 	 * @param array $theme_json The theme.json converted to an array.
-	 * @param array $selectors  Optional list of selectors per block.
 	 *
 	 * @return array The block nodes in theme.json.
 	 */
-	private static function get_block_nodes( $theme_json, $selectors = array() ) {
-		$selectors = empty( $selectors ) ? static::get_blocks_metadata() : $selectors;
+	private static function get_block_nodes( $theme_json ) {
```

Comment:
Removing params like this could be considered a breaking change.

Reply:
Thanks for chiming in. This PR is pretty stale and I was going to abandon it.

After the min version of WordPress Gutenberg requires is 6.1 maybe we could revisit it? 

There's a [PR to bump it to 6.0](https://github.com/WordPress/gutenberg/pull/46102) at the moment.